pkg/controllers/node/node_controller.go

```diff
@@ -15,28 +15,24 @@
 package node
 
 import (
+	"context"
+	"fmt"
 	"reflect"
 	"sync"
-
 	"time"
 
-	calicocache "github.com/projectcalico/kube-controllers/pkg/cache"
-	"github.com/projectcalico/kube-controllers/pkg/controllers/controller"
-	"github.com/projectcalico/libcalico-go/lib/apis/v2"
-	client "github.com/projectcalico/libcalico-go/lib/clientv2"
-
-	"github.com/projectcalico/libcalico-go/lib/errors"
-
 	log "github.com/sirupsen/logrus"
 	"k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/fields"
 	uruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes"
 	corecache "k8s.io/client-go/tools/cache"
 
-	"context"
-	"fmt"
-
+	calicocache "github.com/projectcalico/kube-controllers/pkg/cache"
+	"github.com/projectcalico/kube-controllers/pkg/controllers/controller"
+	api "github.com/projectcalico/libcalico-go/lib/apis/v3"
```

Comment:
is this `apiv3` everywhere else?

Reply:
Might be, but I'd argue we should keep it as `api`, since changing to `apiv3` means more changes, and harder to do if we say, switch to `v4` eventually.

Reply:
Yup my thoughts too.  I was going to fix up a few that were apiv2 to be api but it didn't feel particularly worth the extra effort - better (IMHO) to fix up when we're making "real" modifications to the code.